CHANGELOG.md

```diff
@@ -8,10 +8,20 @@ accidentally triggering the load of a previous DB version.**
 
 **Bugfixes**
 * #4926 Fix corruption when inserting into compressed chunks
+
+## 2.9.2 (2023-01-18)
+
+This release contains bug fixes since the 2.9.1 release.
+We recommend that you upgrade at the next available opportunity.
+
+**Bugfixes**
+* #5180 Fix dist_cagg flaky test
```

Comment:
Could you check the ordering of the entries? It looks like they are not sorted correctly.

Reply:
Right, now ordered it correctly.